keras/utils/data_utils.py

```diff
@@ -299,6 +300,11 @@ def validate_file(fpath, file_hash, algorithm='auto', chunk_size=65535):
         return False
 
 
+def prepare_sliced_sparse_data(data, batch_size):
```

Comment:
a unit test for this function? what if `data.shape[0] < batch_size` or `data.shape[0] is None`? maybe a warning before returning an empty list

Reply:
added